onebusaway-android/src/main/java/org/onebusaway/android/util/RegionUtils.java

```diff
@@ -277,6 +279,51 @@ public static boolean isRegionUsable(ObaRegion region) {
     }
 
     /**
+     *
+     * @return OTP server URL with trimmed trailing slash, or null if none defined for region.
+     */
+    public static String getOtpBaseUrl() {
```

Comment:
Could you please change the method name and signature to:

`public static String formatOtpBaseUrl(String baseUrl) {`

...and don't reference `Application.get()` internally?

In other words, the utility method should do an operation on the string.  I've been trying to keep the Utils classes/methods free from internal statefulness.  This also makes it simpler to write unit tests for various OTP URL formats.